From 1b5bb68e6e10ca11e544661f85f94f949bbb9494 Mon Sep 17 00:00:00 2001
From: Nico Weber <thakis@chromium.org>
Date: Wed, 12 May 2021 22:29:32 +0000
Subject: [PATCH] build: Convert third_party/closure_compiler/compile_js.gni to
 py3

Bug: 1205619
Change-Id: If36551ff1fbf033aab694380157a0f54d2055862
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/2892987
Auto-Submit: Nico Weber <thakis@chromium.org>
Commit-Queue: Dirk Pranke <dpranke@google.com>
Reviewed-by: Dirk Pranke <dpranke@google.com>
Cr-Commit-Position: refs/heads/master@{#882331}
---
 src/third_party/closure_compiler/compile_js.gni | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/src/third_party/closure_compiler/compile_js.gni b/src/third_party/closure_compiler/compile_js.gni
index ae4d16f7ee..82c29dd89b 100644
--- a/src/third_party/closure_compiler/compile_js.gni
+++ b/src/third_party/closure_compiler/compile_js.gni
@@ -2,8 +2,6 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-import("//build/config/python.gni")
-
 import("//third_party/closure_compiler/closure_args.gni")
 
 declare_args() {
@@ -55,8 +53,7 @@ polymer_externs = "$externs_path/polymer-1.0.js"
 #   }
 
 template("js_library") {
-  # TODO(crbug.com/1112471): Get this to run cleanly under Python 3.
-  python2_action(target_name) {
+  action(target_name) {
     script = "$script_path/js_library.py"
     forward_variables_from(invoker,
                            [
@@ -156,8 +153,7 @@ template("js_library") {
 #   }
 
 template("js_binary") {
-  # TODO(crbug.com/1112471): Get this to run cleanly under Python 3.
-  python2_action(target_name) {
+  action(target_name) {
     script = "$script_path/js_binary.py"
     forward_variables_from(invoker,
                            [
